web-service/tests.py

```diff
@@ -14,15 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import argparse
 import json
+import os
+import subprocess
 import urllib
 import urllib2
 
-################################################################################
 
-LOCAL_ENDPOINT = 'http://localhost:8080'
-REMOTE_ENDPOINT = 'http://url-caster.appspot.com'
-REMOTE_DEV_ENDPOINT = 'http://url-caster-dev.appspot.com'
+LOCAL_TEST_PORT = 9002
```

Comment:
Seems we don't document this at all ([here](https://github.com/google/physical-web/tree/master/web-service#how-to-run-your-own-physical-web-service)).

These are the command I usually use:

To run local development server (from web-service directory):
`dev_appserver.py --host 0.0.0.0 --port 8080 --clear_datastore=yes .`

To deploy dev PWS (from web-service directory):
`appcfg.py --oauth2 --no_cookies -A url-caster-dev update .`

To deploy prod PWS (from web-service directory):
`appcfg.py --oauth2 --no_cookies -A url-caster update .`